src/styles/analytics.css

```
@@ -6,15 +6,14 @@
 }
 
 .analytics-header {
-    font-size: 14px;
-    font-weight: 700;
+    display: flex;
+    align-items: center;
+
     border-bottom: 1px solid #ffffff0f;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)